src/main/java/uk/gov/hmcts/darts/common/entity/ArmRpoExecutionDetailEntity.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.darts.common.entity.base.MandatoryCreatedModifiedBaseEntity;
+
+@Entity
+@Table(name = "arm_rpo_execution_detail")
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class ArmRpoExecutionDetailEntity extends MandatoryCreatedModifiedBaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "ard_seq")
+    @SequenceGenerator(name = "ard_seq", sequenceName = "ard_seq", allocationSize = 1)
+    @Column(name = "ard_id", nullable = false)
+    private Integer id;
+
+    @ManyToOne
```

Comment:
Do we want this is to be eager loaded so the state description can be logged rather than the id: fetch = FetchType.EAGER

Reply:
This is the case - `FetchType.EAGER` is the default for `@ManyToOne`.